packages/fleet/lib/fleet.ts

```diff
@@ -0,0 +1,19 @@
+import type { Result } from '@nangohq/utils';
+import { DatabaseClient } from './db/client.js';
+import * as deployments from './models/deployments.js';
+import type { CommitHash, Deployment } from './types.js';
+
+export class Fleet {
+    private dbClient: DatabaseClient;
+    constructor({ fleetId, dbUrl }: { fleetId: string; dbUrl: string }) {
+        this.dbClient = new DatabaseClient({ url: dbUrl, schema: fleetId });
```

Comment:
why not passing the DBClient directly?

Reply:
because I want the db schema to reflect the name/id of the fleet. I could set the schema for the DBClient and then pass the client but it is more encapsulated this way

Reply:
not planning to have multiple fleet anytime soon no but nothing would prevent it in the code